corehq/apps/consumption/shortcuts.py

```diff
@@ -85,30 +84,26 @@ def _update_or_create_default(domain, amount, default, type, **kwargs):
         default.save()
         return default
     else:
-        default = DefaultConsumption(domain=domain, default_consumption=amount, type=type, **kwargs)
+        default = SQLDefaultConsumption(domain=domain, default_consumption=amount, type=type, **kwargs)
         default.save()
         return default
 
 
-def hashable_key(key):
-    """
-    Convert the key from couch into something hasable.
-    Mostly, just need to make it a tuple and remove the special
-    {} value.
-    """
-    return tuple('{}' if item == {} else item for item in key)
-
-
 def build_consumption_dict(domain):
     """
-    Takes raw rows from couch and builds a dict to 
+    Takes raw rows from couch and builds a dict to
     look up consumption values from.
     """
-    raw_rows = get_domain_monthly_consumption_data(domain)
+    SQLDefaultConsumption.objects.filter(domain=domain)
 
     return dict(
-        (hashable_key(row['key']), Decimal(row['value']))
-        for row in raw_rows if row['value']
+        (tuple((
```

Comment:
Wait, `tuple` is not necessary. Comma-separated items (often wrapped in parens) represent a tuple-literal.
```suggestion
        (((
```

Could also get rid a superfluous set of parens.